### DISCUSSION TITLE
Is it possible to use type-to-nav mode without automatically entering directories?

### DISCUSSION THREAD

**Discussion by ezhang7423** [jarun/nnn#1705] [Q&A]
I love the type-to-nav mode, but oftentimes I get confused when it automatically jumps into a new directory when there is only one filtered option left. Is it possible to configure it to disable this behavior, and require pressing enter first?

**Comment by N-R-K** [comment]
See the manpage. The `-A` flag should do what you want.

**Reply by ezhang7423** [reply]
Thank you, not sure how I missed this!
